### PR TITLE
DM-7229: Pytest updates

### PR DESCRIPTION
**Pull request by timj**

### CODE REVIEW

File: tests/testImportCentroidAlgorithm.py [hunk 1]
```diff
@@ -59,11 +60,11 @@ def testApplyCentroid(self):
         """Test that we can instantiate and play with SillyMeasureCentroid"""
 
         for imageFactory in (
-                             afwImage.MaskedImageF,
-                             ):
+            afwImage.MaskedImageF,
```
Comment (timj) [inline tests/testImportCentroidAlgorithm.py]: I wonder why this is on a separate line. Were there multiple image types in this test originally?

Comment (isullivan) [inline tests/testImportCentroidAlgorithm.py]: I strongly considered condensing it to one line, but assumed the designer anticipated adding more in the future. I'd be happy to change it.

File: tests/testMeasureSources.py [hunk 2]
```diff
@@ -295,7 +291,7 @@ def testPixelFlags(self):
         source.set("centroid_y", 40)
         source.set("centroid_flag", True)
         source.setFootprint(afwDetection.Footprint(afwGeom.Point2I(afwGeom.Point2D(x + x0, y + y0)), 5))
-        self.assertRaises(  
+        self.assertRaises(
```
Comment (timj) [inline tests/testMeasureSources.py]: @r-owen has suggested we rewrite these using:
```python
with self.assertRaises(lsst.pex.exceptions.RuntimeError):
    plugin.measure(source, exp)
```
I agree with him that this looks much more readable and obvious.

File: tests/testImportCentroidAlgorithm.py (hunk 1 above)
Comment (timj) [inline tests/testImportCentroidAlgorithm.py]: Leave it be :-)

**Pull request merged**